noxfile.py

```diff
@@ -173,7 +175,6 @@ def unit_tests(session: nox.Session) -> None:
         "-r",
         "requirements_dev.txt",
         # For test_reports:
```

Comment:
If you removed the dependency could you also remove the comment?

Reply:
it still applies to the next line

Reply:
(polyfactory)

Reply:
so why are you deleting that dependency?

Reply:
which one? .[reports] does not exist anymore as it needed pydantic, which is now in package deps. polyfactory is only needed for their tests, so it makes sense for it to be here.

Reply:
yeah i didn't look what reports include, got it thanks